EpiserverRedirects/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -21,12 +20,6 @@ public static IApplicationBuilder UseEpiserverRedirects(this IApplicationBuilder
                 redirectsEventsRegistry.RegisterEvents();
             }
 
-            if (redirectsOptions.Caching.AllRedirectsCacheEnabled || redirectsOptions.Caching.UrlRedirectCacheEnabled)
```

Comment:
OK, now I remember what was the case - we cannot solely rely on DDS events now as we might have different types of repositories. But we still need a way of clearing RedirectRules cache in case something new is inserted/updated/deleted.
We will probably need some wrapper for IRepository as everything goes directly through it as of now 🤔